src/Run.hs

```diff
@@ -44,6 +44,7 @@ parseOptions progName args = case args of
   ["--list-installable"] -> ListInstallable
   ["--list-bootstrapped"] -> ListBootstrapped
   ["--bootstrap", packageSetName] -> Bootstrap $ PackageSetName packageSetName
+  (invalid : invalids) | "-" `isPrefixOf` invalid && "--help" `elem` invalids -> Help (helpText progName)
```

Comment:
This fails for `(words "--help --invalid")`. I guess the cases for `--help` should be merged somehow.